overlay.d/05rhcos/usr/libexec/rhcos-fix-root

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -euxo pipefail
+
+if [ -f /run/device_mod ]; then
```

Comment:
I think we can drop the `/run/device_mod` stamp file if we use the `ConditionKernelCommandLine=rhcos.root` bit?

Reply:
I asked for that stamp.  The concern is that the script should not run twice if it's started twice within the same boot.